Engines/Wine/QuickScript/Zip Script/script.js

```diff
@@ -55,9 +67,16 @@ module.default = class ZipScript extends QuickScript {
         new Extractor()
             .wizard(setupWizard)
             .archive(archive)
-            .to(wine.prefixDirectory() + "/drive_c/" + this._name)
+            .to(`${wine.prefixDirectory()}/drive_c/${this._name}`)
             .extract();
 
+        if (this._setupPath !== `undefined`) {
```

Comment:
This check does not seems to work with crayon physics. However the variable is `undefined` so what follows should not be executed :thinking: .

Reply:
Should it be
```Javascript
if (this._setupPath !== `undefined`) {
```
or
```Javascript
if (this._setupPath !== undefined) {
```

Reply:
Okay. It works now.